snapcraft/tests/test_commands_list_plugins.py

```diff
@@ -23,25 +24,45 @@
 
 class ListPluginsCommandTestCase(tests.TestCase):
 
+    # plugin list when wrapper at MAX_CHARACTERS_WRAP
+    default_plugin_output = (
+        "ant        catkin  copy  jdk     kernel  maven  nodejs   python3  tar-content\n"  # noqa
+        "autotools  cmake   go    kbuild  make    nil    python2  scons  \n")  # noqa
```

Comment:
For consistency, please use single quotes for strings.
And it hurts me seeing python code with more than 80 lines, I'm weird that way. Would it be too ugly to split the string before the \n?
'ant        catkin  copy  jdk     kernel  maven  nodejs   '
'python3  tar-content\n'
'autotools  cmake   go    kbuild  make    nil    '
'python2  scons  \n'